packages/lib/callbackUrlUtils.ts

```diff
@@ -0,0 +1,37 @@
+const URL = require('url-parse');
```

Comment:
Please add test units for this file. See other .test.ts files to see how it's done.

Reply:
this is done